datafaker-gen-examples/datafaker-gen-bigquery/pom.xml

```diff
@@ -161,4 +161,48 @@
         </plugins>
     </build>
 
+
+    <profiles>
+        <profile>
+            <id>default</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <version>3.3.0</version>
```

Comment:
Could you please move version to properties
like it is done for datafaker https://github.com/datafaker-net/datafaker/blob/f8b2473528c3dbe0a90c835202c3e1bef0d5be33/pom.xml#L46